tools/adventure-pack/goodies/typescript/Iterator.prototype.max/index.test.ts

```diff
@@ -1,5 +1,8 @@
 import { describe, expect, it } from "@jest/globals";
 
+import { iteratorPrototype } from "../Iterator.prototype";
+delete (iteratorPrototype as unknown as Record<string, unknown>).max;
```

Comment:
Can `max` exist as a built-in? I don't think I've seen it on MDN. I think we should only do this for functions that are part of some experimental proposal.

Reply:
Excluded `min` and `max`, and included `some`